gateway/webhook/integration_test.go

```diff
@@ -335,6 +359,20 @@ func TestIntegrationWebhook(t *testing.T) {
 				errPayload, err = sjson.SetBytes(errPayload, "source.Destinations", nil)
 				require.NoError(t, err)
 
+				payload := gjson.GetBytes(r.Jobs[i].EventPayload, "request.body").String()
+				fmt.Println("payload", payload)
```

Comment:
Is this log useful for debugging the test in case it fails?
```suggestion
				t.Log("payload", payload)
```

Reply:
This is not necessary. Cleaning this up.